packages/redux/src/users/actions/updateUser.js

```diff
@@ -0,0 +1,51 @@
+import {
+  UPDATE_USER_FAILURE,
+  UPDATE_USER_REQUEST,
+  UPDATE_USER_SUCCESS,
+} from '../actionTypes';
+
+/**
+ * @callback UpdateUserThunkFactory
+ * @param {number} id - User identifier.
+ * @param {object} data - User data.
```

Comment:
Same as my previous comment.

Reply:
The JSDoc will be review in the Typescript phase 1 task, will add it then.